app/shared/resource-type-filter/ResourceTypeFilterButton.js

```diff
@@ -0,0 +1,23 @@
+import classnames from 'classnames';
+import React, { PropTypes } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+
+function ResourceTypeFilter({ active, onClick, resourceType }) {
+  return (
+    <Button
+      bsStyle={classnames({ primary: active, default: !active })}
+      className="resource-type-button"
+      onClick={() => onClick(resourceType)}
+    >
+      { resourceType }
```

Comment:
- [ ] You could also capitalize the resourceType text so all the texts in the buttons look the same.